tests/test_resolvers/test_join.py

```diff
@@ -0,0 +1,30 @@
+from unittest.mock import Mock
+
+from sceptre.resolvers import Resolver
+from sceptre.resolvers.join import Join
+
+
+class FirstResolver(Resolver):
+    def resolve(self):
+        return "first"
+
+
+class SecondResolver(Resolver):
+    def resolve(self):
+        return "last"
+
+
+class TestJoin:
```

Comment:
would it make sense to test when a resolver returns non-string values to join? also a test when resolver returns None?

Reply:
Great suggestions. I suppose in the first case, I'll want it to coerce the values to strings. In the second case, they'll just be removed from the list. I'll add those tests!

Reply:
Done!